users/reverseproxy.rst

```diff
@@ -39,6 +40,7 @@ Nginx
 
     location /syncthing/ {
       proxy_set_header        Host $host;
+      #proxy_set_header        Host localhost; # Uncomment to bypass Host Check
```

Comment:
Probably wants a comment saying to remove `proxy_set_header        Host $host;` if you do this?